### PR TITLE
Differential gene expression app

### PR DESCRIPTION
**Pull request by jordanvelezbomba**

### CODE REVIEW

File: screen2.0/src/app/applets/differential-gene-expression/utils.tsx [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+    start: 108107280,
+    end: 108146146,
+    id: "ENSMUSG00000000001.4",
+    name: "Gm25142",
+}
+
+/**
+ * define types for list of cell types
+ */
+export const initialCellTypes = {
```
Comment (NishiPhalke) [inline screen2.0/src/app/applets/differential-gene-expression/utils.tsx:312]: Are we using this anywhere ?

**Review by NishiPhalke — COMMENTED** [review]
Code Looks good. Few minor comments:

**Review by NishiPhalke — COMMENTED** [review]
Code Looks good. Few minor comments:

function paramters in de utils can be types
can we move few constants from utils to consts.ts in de app (for e.g initialGeneList,payload, initialCellTypes, etc.)
Files where we are not returning any tsx/jsx (html like) elements can be named .ts instead of .tsx (for e.g utils.ts, conts.ts, colors.ts etc.)

**Review by NishiPhalke — APPROVED** [review]

File: screen2.0/src/app/applets/differential-gene-expression/utils.tsx (hunk 1 above)
Comment (jordanvelezbomba) [reply screen2.0/src/app/applets/differential-gene-expression/utils.tsx:312]: don't think so, will delete

**Pull request merged**